public_api/src/machine_registry_internal.ts

```diff
@@ -69,3 +73,8 @@ export interface MangledOnButtonPressedPayload {
    */
   d: string;
 }
+
+export interface MangledNetworkStatsEventArg {
```

Comment:
the name says "mangled" but the property names are not mangled.
```suggestion
export interface IpcNetworkStatsEventArg {
```